smartsim/_core/control/job.py

```diff
@@ -25,8 +25,10 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import time
+import typing as t
```

Comment:
as far as I can see you're not using this import

Reply:
good 👁️ !